docs/whats-new.rst

```diff
@@ -51,6 +51,12 @@ Bug Fixes
   writing to netCDF files to avoid bugs when using ``libnetcdf``
   version 4.5.0 (:pull:`235`).  By `Spencer Hill
   <https://github.com/spencerahill>`_.
+- In ``CalcSuite`` when calling ``submit_mult_calc``, calculations
```

Comment:
Other way around: `submit_mult_calc` calls `CalcSuite`.  How about "`CalcSuite` (and thus `submit_mult_calcs`) now skips calculations that involve..."